src/main/java/mate/controller/AddDriverForCarController.java

```diff
@@ -0,0 +1,32 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/add")
+public class AddDriverForCarController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("mate");
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = Long.parseLong(req.getParameter("driverId"));
+        Long carId = Long.parseLong(req.getParameter("carId"));
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
```

Comment:
move carService and driverService up to the class variables. it is enough to create one instance for each service class for the whole time while application working.
please fix this suggestion for all controllers below.